### PR TITLE
Fix deprecated rule in .golangci.yml

### PR DESCRIPTION
**Pull request by pysel**
## 1. Summary
Fixes #None

## 2.Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## 5. Checklist

- [ ] Does the Readme need to be updated?

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -2,8 +2,6 @@ run:
   tests: true
   #   # timeout for analysis, e.g. 30s, 5m, default is 1m
   timeout: 10m
-  skip-dirs:
-    - third-party-chains
```
Comment (pysel) [inline .golangci.yml:6]:
```
WARN [config_reader] The configuration option `run.skip-dirs` is deprecated, please use `issues.exclude-dirs`. 
```

Comment (pysel) [reply .golangci.yml:6]:
```
ruslanakhtariev@169-231-72-96 quicksilver % golangci-lint version                         
golangci-lint has version 1.57.2 built with go1.22.1 from 77a8601a on 2024-03-28T19:01:11Z
```

**Review by faddat — APPROVED** [review]

**Review by faddat — CHANGES_REQUESTED** [review]
Hey, it seems that this has strangely angered the winter. Could you check out what the issue is?

**Review by joe-bowman — APPROVED** [review]
lgtm

**Comment by joe-bowman** [comment]
I believe the issue here is that .github/workflows/golangci.yaml is using 1.56 not 1.57 and the setting while valid for your local instance, is not valid for the CI. Updating the CI version to 1.57.2 should do the trick, thank you!

**Pull request merged**